crowbar_framework/vendor/assets/stylesheets/mixins/_keyframes.scss

```diff
@@ -0,0 +1,14 @@
+@mixin keyframes($animation-name) {
+  @-webkit-keyframes #{$animation-name} {
+    @content;
```

Comment:
This is a very cool feature form sass.. less still doesnt have anything like it. :+1: